src/azul/service/hca_response_v5.py

```diff
@@ -248,17 +248,26 @@ def make_bundles(self, entry):
         return [{"bundleUuid": b["uuid"], "bundleVersion": b["version"]} for b in entry["bundles"]]
 
     def make_protocols(self, entry):
-        protocols = []
-        for protocol in entry["contents"]["protocols"]:
-            translated_process = {
-                "libraryConstructionApproach": protocol.get("library_construction_approach", [None]),
-                "instrumentManufacturerModel": protocol.get("instrument_manufacturer_model", [None]),
-                "pairedEnd": protocol.get("paired_end", [None]),
-                "workflow": protocol.get("workflow", [None]),
-                "assayType": protocol.get("assay_type", [None]),
-            }
-            protocols.append(translated_process)
-        return protocols
+        return (
+            [
+                {
+                    'workflow': protocol.get('workflow', None)
+                } for protocol in entry['contents']['analysis_protocols']
+            ] + [
```

Comment:
Conventionally, we've been using interpolated generator expressions for this structure:

`[*({…} for …), *({…} for …), …]`

Reply:
We could also use `itertools.chain(({…} for …), ({…} for …), …)`

but I think we should stick with precedent even though I don't really know which is more efficient. And if it even matters. What you wrote looks good but I'm worried it actually makes four lists and then concatenates two and then two more etc. I don't want to go down the micro-optimization rabbit hole so I think precedent (using `*()`) wins.